3.resistance-signature/scripts/nbconverted/8.1. last-batch-compile-otherclone-datase.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Compile dataset of clones resistant to other drugs
+# 
+# **Gregory Way, 2021**
+# 
+# **Yu Han, 2021**
+# 
+# This script is modified from Greg Way's original scripts of 8.compile-otherclone-dataset.
+# 
+# This dataset includes new batches of 24~27 including WT (10, 12-15) and Bz-resistant (6-10) clones.
+# They are DMSO-treated clones to later apply the bortezomib resistance signature.
+
+# In[1]:
+
+
+import sys
+import pathlib
+import numpy as np
+import pandas as pd
+
+from pycytominer.cyto_utils import infer_cp_features
+
+sys.path.insert(0, "../2.describe-data/scripts")
+from processing_utils import load_data
+
+
+# In[2]:
+
+
+np.random.seed(1234)
+
+
+# In[3]:
+
+
+data_dir = pathlib.Path("../0.generate-profiles/profiles")
+cell_count_dir = pathlib.Path("../0.generate-profiles/cell_counts/")
+
+output_dir = pathlib.Path("data")
+
+profile_suffix = "normalized.csv.gz"
+
+
+# In[4]:
+
+
+datasets_val = {
```

Comment:
Just to confirm, these are the only plates in these new batches that include WT and BZ clones treated together on the same plate with a treatment of 0.1% DMSO?

If confirmed already, sounds good! Please add a comment to the corresponding `.ipynb` notebook in this cell describing why these plates in particular are important.

Reply:
Yes, these are the only plates that have the identified WT and BZ clone types treated with 0.1% DMSO. My understanding for their importance is that these clone types are either sensitive or resistance to bortezomib. Clones A and E as resistant to bortezomib, and the WT parental would score somewhere in the "sensitive to bortezomib" range, since they are a heterogenous population but should mostly be composed of sensitive cells.